### DISCUSSION TITLE
World persistence

### DISCUSSION THREAD

**Discussion by tmcgroul** [yeet-zone/docker-dontstarvetogether#154] [Q&A] [answered]
Hi, i ran a container with volumes then i removed the container and after i started another container with volumes from previous one. I expected that a server would be started from previous state but it generated a new world.
So i have a question: can i start a new container from existed state?

**Comment by thasmo** [comment, marked as answer]
@tmcgroul, yes, this is possible. It seems I've removed the relevant configurations by accident when updating and refactoring the `recipes` documentation. Have a look at the [`session`](https://github.com/yeetzone/docker-dontstarvetogether/blob/master/docs/recipes/session/) recipe, which [configures a separate volume](https://github.com/yeetzone/docker-dontstarvetogether/blob/master/docs/recipes/session/docker-compose.yml#L18-L20) for the server's `storage` directory so that even when destroying and recreating the docker containers from scratch, the world and the characters, etc. is persisted via the volume.

**Reply by tmcgroul** [reply]
It works!!!
Thanks for your help
